examples/todos/lib/todo.dart

```diff
@@ -5,7 +5,7 @@ const _uuid = Uuid();
 
 /// A read-only description of a todo-item
 class Todo {
-  Todo({
+  const Todo({
```

Comment:
Could you add `@immutable` then?

Reply:
Yes sounds like a good idea, wonder why I didn't do that in the first place. Will fix soon...